### PR TITLE
Fix/Camera and storage permission

### LINKED ISSUE #194: Denied permissions hides attachment button permanently
And granted permission from app settings doesn't make attachment button visible again:
Mon_Oct_21_01:22:56_CEST_2019

### PR DESCRIPTION
**Pull request by adrian09h**
# Submit a pull request
close #194

### CODE REVIEW

**Comment by adrian09h** [comment]
> If fixes the issue, but introduces another: when system asks permission and user selects "Never ask" attachment still visible, but when user presses it again nothing happens.
> 
> That's not how it should be implemented. When permission request is blocked app should show message with button which leads to permissions settings. Check for the reference `Telegram` implementation.

I'm agreement with you, I will fix it soon like `Telegram`

File: library/src/main/java/com/getstream/sdk/chat/view/MessageInputView.java [hunk 1]
```diff
@@ -393,6 +406,29 @@ private boolean isGrantedPermissions() {
             return true;
     }
 
+    private void showPermissionSettingDialog(){
+        String appName = Utils.getApplicationName(getContext());
+        String msg = appName + getContext().getString(R.string.stream_camera_permission_message);
```
Comment (elevenetc) [inline library/src/main/java/com/getstream/sdk/chat/view/MessageInputView.java]: The dialog shows `camera` permission issue only. If user allowed `camera`, but blocked `storage` the message will still be about `camera`. So we either need to show correct message passing non granted permission or just show generic message about permissions

Comment (adrian09h) [reply library/src/main/java/com/getstream/sdk/chat/view/MessageInputView.java]: @elevenetc ,exactly. currently the branch was marked as wip. :) work in progress yet.

**Comment by adrian09h** [comment]
@elevenetc updated everything according to your comments.

**Pull request merged**